plugin.php

```diff
@@ -573,12 +577,25 @@ function rest_handle_options_request( $response, $handler, $request ) {
 
 		foreach ( $endpoints as $endpoint ) {
 			$accept = array_merge( $accept, $endpoint['methods'] );
+
+			$methods = array_keys( $endpoint['methods'] );
+			$method = array_shift( $methods );
+
+			if ( empty( $body['request_args'][ $method ] ) ) {
+				$body['request_args'][ $method ] = $endpoint['args'];
+			}
+
+			if ( empty( $body['schema'] ) && ! empty( $endpoint['schema'] ) ) {
```

Comment:
Hacky thing to default to the first time the schema was registered for a resource.